test/unit/math/prim/mat/fun/multiply_test.cpp

```diff
@@ -133,3 +133,31 @@ TEST(AgradRevMatrix, multiply_vector_int) {
   EXPECT_EQ(4.0, prod_vec[1]);
   EXPECT_EQ(6.0, prod_vec[2]);
 }
+
+#ifdef STAN_OPENCL
+#define EXPECT_MATRIX_NEAR(A, B, DELTA) \
+  for (int i = 0; i < A.size(); i++)    \
+    EXPECT_NEAR(A(i), B(i), DELTA);
+
+TEST(MathMatrix, multiply_opencl) {
+  int multiply_dim_prod_worth_transfer
+      = stan::math::opencl_context.tuning_opts()
+            .multiply_dim_prod_worth_transfer;
```

Comment:
Why is this here?

Reply:
goof. Used to be two parameter and I replaced them real quick.

Reply:
Ignore the above comment.

Its there so we set the tuning parameter back to the original value that it was before the test. So that the next test case (there isnt one currently, but if someone does add one) doesnt have to manually re-set it. 

simple case:
```cpp
  stan::math::matrix_d m3 = multiply(m1, m2);

  EXPECT_MATRIX_NEAR(m3_cl, m3, 1e-10);
  stan::math::opencl_context.tuning_opts().multiply_dim_prod_worth_transfer
      = multiply_dim_prod_worth_transfer;
  stan::math::opencl_context.tuning_opts().multiply_dim_prod_worth_transfer
      = 42;
}

TEST(MathMatrix, multiply_opencl1) {
  // this will output 42 which is bad
  std::cout << stan::math::opencl_context.tuning_opts().multiply_dim_prod_worth_transfer << std::endl;
}
```